### ISSUE TITLE
Invalid oktaAuth userAgent mutation breaks version check

### ISSUE THREAD

**Issue by theseyi** [okta/okta-react#162]
@oleksandrpravosudko-okta 
This change has introduced a pretty annoying bug that manifests when the app is reloaded via HMR. not sure why `oktaAuth` needs to be mutated to add the okta-react as userAgent but there will now be two agents when this [runs](https://github.com/okta/okta-react/blob/5642b55943ecedcafebb7d3177026ac7f8282e9c/src/Security.tsx#L54):
```
oktaAuth.userAgent
"@okta/okta-react/6.1.0 okta-auth-js/5.3.1"
```
This will then break the check for `oktaAuthMajorVersion` on [line 85](https://github.com/okta/okta-react/blob/5642b55943ecedcafebb7d3177026ac7f8282e9c/src/Security.tsx#L85) previously added by @shuowu because `oktaAuthMajorVersion` will now evaluate to `"okta-react"` whereas `process.env.AUTH_JS_MAJOR_VERSION ` evaluates to `5`... `"okta-react" === 5 -> false`
causing
```tsx
    return <OktaError error={err} />;
```

~_Originally posted by @theseyi in https://github.com/okta/okta-react/pull/159#r697899944_~

UPDATED to correct references

**Comment by theseyi** [comment]
Nvm, I see this was actually the fix for the issue 😅  Since this is a hotfix, any ETA on release date?

**Comment by shuowu** [comment]
@theseyi #159 has been released in version 6.2.
